Gemfile

```diff
@@ -15,6 +15,9 @@ gem 'puma'
 # Docker
 gem 'tzinfo-data'
 
+# Misc
+gem 'awesome_print'
```

Comment:
Why do we need that gem?

Reply:
I use it frequently for debugging -- the nice thing about it is that it can print just about anything you hand it, object, hashes, lists, etc.

Reply:
But I don't think it should be generally required to install it. Maybe put it in development group?